tests/test_other.py

```diff
@@ -6844,12 +6867,12 @@ def test_python_2_3(self): # check emcc/em++ can be called by any python
       print(python, has)
       if has:
         print('  checking emcc...')
-        check_execute([python, EMCC, '--version'])
+        check_execute([python, EMCC[:-3], '--version'])
```

Comment:
shouldn't it work with or without `.py`? Maybe I don't remember why the test checks the `.py` version later...

Reply:
Yes, it works with or without it. I did this because the test specifically wants to test `emcc`.

Reply:
Could this be a `def remove_py_suffix()` function or similar that looks for `.py` at the end and removes that? This could be a bit brittle otherwise for future refactors in mind, if one unconditionally just removes three last chars here?